Modules/Inventory/inventory.gd

```diff
@@ -116,6 +116,6 @@ func remove(item: Item, amount: int):
 func contains(item: Item, amount: int):
 	var amount_has: int = 0
 	for i in range(inv.size()):
-		if(inv[i].item == item):
```

Comment:
this crashes when checking a blank inv slot, added an extra check to prevent that